Gemfile.lock

```diff
@@ -45,8 +44,7 @@ GEM
     method_source (1.0.0)
     mini_mime (1.1.5)
     minitest (5.18.0)
-    multi_xml (0.7.1)
-      bigdecimal (~> 3.1)
```

Comment:
Updated `httparty` which depends on a version of `multi_xml` that does not exist anymore.

> Your bundle is locked to multi_xml (0.7.1) from rubygems repository
https://rubygems.org/ or installed locally, but that version can no longer be
found in that source. That means the author of multi_xml (0.7.1) has removed it.

Reply:
Awesome! Thanks for fixing that!